src/styles/main.css

```
@@ -75,18 +91,26 @@ button.primary {
   width: 100%;
   height: 100%;
   z-index: 1;
+  overflow: hidden;
+}
+
+.top-header-conatiner {
+  display: flex;
+  padding-left: 250px;
+  padding-top: 48px;
+
+  justify-content: center;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)